Outfit Selection/View Controllers/Main/OutfitViewController/OutfitViewController.swift

```diff
@@ -51,6 +51,13 @@ class OutfitViewController: LoggingViewController {
     /// Confstraint for the trailing of hanger bubble
     var hangerBubbleTrailingConstraint: NSLayoutConstraint!
     
+    /// Current status loading occasions elements
+    var occasionItemsAreLoading = false {
+        didSet {
+            debug("Status occasionsItemsIsLoading was changed to \(occasionItemsAreLoading)" )
```

Comment:
Replace with `debug("DEBUG: occasionsItemsAreLoading changed to \(occasionItemsAreLoading)" )`